src/main/java/lotto/model/Receipt.java

```diff
@@ -0,0 +1,55 @@
+package lotto.model;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class Receipt {
+    private final Money money;
+    private Ticket ticket;
+    private Lotteries auto;
+    private Lotteries manual;
+
+    public Receipt(Money input) {
```

Comment:
```suggestion
    public Receipt(Ticket ticket, Lotteries manual) {
```
이렇게 받아서 manual 개수만큼 ticket을 줄이고, 해당 ticket을 LotteryStore.exchangeToLotteries(ticket) 으로 보내면 
ticket 필드를 없애고, auto와 manul을 불변 객체(final)로 다룰 수 있지 않을까요...?